fields/civimembershiplevels.php

```diff
@@ -15,72 +15,70 @@
 /**
  * Form Field class for the Joomla Framework.
  *
- * @package		Joomla.Framework
- * @subpackage	Form
- * @since		1.6
+ * @package      Joomla.Framework
+ * @subpackage   Form
+ * @since        1.6
  */
-class JFormFieldCiviMembershipLevels extends JFormField
-{
-	/**
-	 * The form field type.
-	 *
-	 * @var		string
-	 * @since	1.6
-	 */
-	protected $type = 'List';
-	var	$_name = 'CiviMembershipLevels';
+class JFormFieldCiviMembershipLevels extends JFormField {
+  /**
+   * The form field type.
+   *
+   * @var      string
+   * @since    1.6
+   */
+  protected $type = 'List';
+  var    $_name = 'CiviMembershipLevels';
 
-	/**
-	 * Method to get the field input markup.
-	 *
-	 * @return	string	The field input markup.
-	 * @since	1.6
-	 */
-	protected function getInput()
-	{
-		// Initialize variables.
-		$html = array();
-		$attr = '';
+  /**
+   * Method to get the field input markup.
+   *
+   * @return   string    The field input markup.
+   * @since    1.6
+   */
+  protected function getInput() {
+    // Initialize variables.
+    $html = array();
+    $attr = '';
 
-		// Initiate CiviCRM
-		require_once JPATH_ROOT.'/'.'administrator/components/com_civicrm/civicrm.settings.php';
-		require_once 'CRM/Core/Config.php';
-		$config =& CRM_Core_Config::singleton( );
+    // Initiate CiviCRM
+    require_once JPATH_ROOT . '/' . 'administrator/components/com_civicrm/civicrm.settings.php';
```

Comment:
go ahead and combine the forward slash here too

Reply:
ok